packages/go_router/example/lib/main.dart

```diff
@@ -1,93 +0,0 @@
-// Copyright 2013 The Flutter Authors. All rights reserved.
```

Comment:
I remember the main.dart will be displayed on the pub site https://pub.dev/packages/go_router/example
we should probably keep this